Gemfile.lock

```diff
@@ -121,36 +122,35 @@ GEM
       activesupport (>= 3.0.0)
       uniform_notifier (~> 1.11)
     byebug (11.1.3)
-    capistrano (3.14.0)
+    capistrano (3.14.1)
       airbrussh (>= 1.0.0)
       i18n
       rake (>= 10.0.0)
       sshkit (>= 1.9.0)
-    capistrano-bundler (1.6.0)
+    capistrano-bundler (2.0.1)
       capistrano (~> 3.1)
-    capistrano-rails (1.5.0)
+    capistrano-rails (1.6.1)
       capistrano (~> 3.1)
-      capistrano-bundler (~> 1.1)
+      capistrano-bundler (>= 1.1, < 3)
     capistrano-rails-console (2.3.0)
       capistrano (>= 3.5.0, < 4.0.0)
       sshkit-interactive (~> 0.3.0)
     case_transform (0.2)
       activesupport
-    code_analyzer (0.5.1)
+    code_analyzer (0.5.2)
       sexp_processor
     codeclimate-engine-rb (0.4.1)
       virtus (~> 1.0)
     coderay (1.1.3)
     coercible (1.0.0)
       descendants_tracker (~> 0.0.1)
-    concurrent-ruby (1.1.6)
-    crack (0.4.3)
-      safe_yaml (~> 1.0.0)
+    concurrent-ruby (1.1.7)
+    crack (0.4.4)
     crass (1.0.6)
     debug_inspector (0.0.3)
     descendants_tracker (0.0.4)
       thread_safe (~> 0.3, >= 0.3.1)
-    devise (4.7.1)
+    devise (4.7.3)
```

Comment:
### [devise](https://github.com/heartcombo/devise)

**Patch** version upgrade :chart_with_upwards_trend::small_blue_diamond: 4.7.1 → 4.7.3

[_~~change-log~~, [source-code](http://github.com/plataformatec/devise)_]





<details>
<summary>Commits</summary>

A change of **93** commits. See the full changes on [the compare page](https://github.com/heartcombo/devise/compare/v4.7.1...v4.7.3).

These are the first 10 commits:
- (a823e51) [Using scoped errors for scoped views. Fixes #5066](https://github.com/heartcombo/devise/commit/a823e510f3988be53b74eae9ae28af1084a23005)
- (63ea653) [increase default stretches to 12](https://github.com/heartcombo/devise/commit/63ea6533de34b6457b31d375fb30cd44d6403616)
- (45245df) [update changelog](https://github.com/heartcombo/devise/commit/45245df16a6d617d21194dfa398de9ceedfc56f8)
- (b52e642) [Merge pull request #5074 from sergey-alekseev/increase-defau…](https://github.com/heartcombo/devise/commit/b52e642c0131f7b0d9f2dd24d8607a186f18223e)
- (34ed989) [Move PR #5074 to unreleased in changelog [skip ci]](https://github.com/heartcombo/devise/commit/34ed98972518d035b28d359e774aa6e0608c0979)
- (f48b6f1) [Merge pull request #5067 from shobhitic/master](https://github.com/heartcombo/devise/commit/f48b6f1651233a39c52dd70e9ac6c35f70086062)
- (5d73e1e) [Explain layout of default config initializer [ci skip]](https://github.com/heartcombo/devise/commit/5d73e1e3bb4961e143ca3d6aa75ca13a7ef9a7cd)
- (0f134f7) [Call set_flash_message helper instead of flash accessor](https://github.com/heartcombo/devise/commit/0f134f7030edbca0366f2979e9fbe48abe02edd8)
- (421ffc4) [Add test to admin error message](https://github.com/heartcombo/devise/commit/421ffc479fa83720c23343978200ea6ad1b461f3)
- (d022fb8) [Update code with single-quotes](https://github.com/heartcombo/devise/commit/d022fb8cc4490479421c78b98586463e3978f511)


</details>